.changeset/fuzzy-pumpkins-tie.md

```diff
@@ -0,0 +1,5 @@
+---
+'@backstage/catalog-model': minor
```

Comment:
Maybe this should be patch since it's additive?

Reply:
It's also not a breaking change, so patch should be fine?